bin/i18n/sync-down.rb

```diff
@@ -7,9 +7,14 @@
 require_relative 'i18n_script_utils'
 
 def sync_down
-  system "crowdin --config #{CODEORG_CONFIG_FILE} --identity #{CODEORG_IDENTITY_FILE} download translations"
-  system "crowdin --config #{CODEORG_MARKDOWN_CONFIG_FILE} --identity #{CODEORG_MARKDOWN_IDENTITY_FILE} download translations"
-  system "crowdin --config #{HOUROFCODE_CONFIG_FILE} --identity #{HOUROFCODE_IDENTITY_FILE} download translations"
+  puts "Beginning sync down"
+
+  CROWDIN_PROJECTS.each do |name, options|
+    puts "Downloading translations from #{name} project"
+    system "crowdin --config #{options[:config_file]} --identity #{options[:identity_file]} download translations"
+  end
+
+  puts "Sync down complete"
```

Comment:
Should we check return codes for errors and make sure the messages reflect that?

Reply:
we should! I did that for the sync down in https://github.com/code-dot-org/code-dot-org/pull/31783/files, but I'll also open a PR to do the same for the sync up